var/spack/repos/builtin/packages/dia/package.py

```diff
@@ -34,15 +34,18 @@ class Dia(Package):
 
     depends_on('intltool', type='build')
     depends_on('gtkplus@2.6.0:')
-    depends_on('cairo')
-    depends_on('libpng')
```

Comment:
if it is needed by `dia`, then, indeed, we shall not rely on them being in `RPATH` and should keep these `depends_on()`.